app/src/main/res/layout/fragment_create.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
```

Comment:
As you are already in the fragment_create I would suggest to remove create word from the ids of the views . You can leave it for the create_now_button only.